snmp_cisco/manifest.json

```diff
@@ -3,25 +3,17 @@
   "maintainer": "help@datadoghq.com",
   "manifest_version": "1.0.0",
   "name": "snmp_cisco",
-  "metric_prefix": "snmp.",
-  "metric_to_check": [
-    "snmp.cefcFRUPowerAdminStatus",
-    "snmp.devClientCount"
-  ],
+  "metric_to_check": ["snmp.cefcFRUPowerAdminStatus", "snmp.devClientCount"],
```

Comment:
I think we should avoid some style changes, that will help keep commit history.

Maybe we should lint/format manifest.json using a tool (validation on CI). Not for this PR, but an idea for future.

Reply:
This change is probably due to my vscode configuration `max_line_length`. Happy to set it up with a team configuration for this to not happen anymore

Reply:
I dropped my previous commit, disable formatting on save for `json` files on my vscode settings, and changed only the needed lines. Sorry for the issue.

Reply:
Thanks :)